web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -102,14 +105,19 @@ const FORECAST_WARN_MESSAGE = 'A forecast cannot contain N/A values.'
 const MoreCast2Page = () => {
   const classes = useStyles()
   const dispatch: AppDispatch = useDispatch()
-  const { fireCenters } = useSelector(selectFireCenters)
+  const { groups, loading: groupsLoading } = useSelector(selectStationGroups)
+  const tableLoading = useSelector(selectMorecast2TableLoading)
+  const { members } = useSelector(selectStationGroupsMembers)
```

Comment:
Could we sort the members alphabetically somewhere? They display alphabetically in WF1 groups.

Reply:
Maybe in stationAPI.ts?